File: CampingNeretva/CampingNeretva.Model/ReservationAccommodationModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CampingNeretva.Model
{
    public class ReservationAccommodationModel
    {
        //public int AccommodationId { get; set; }
        public int Quantity { get; set; }
        public AccommodationModel Accommodation { get; set; }
    }
}

File: CampingNeretva/UI/campingneretva_desktop/lib/models/reservation_accommodation_model.dart
import 'accommodation_model.dart';

class ReservationAccommodationModel {
  final int quantity;
  final Accommodation accommodation;

  ReservationAccommodationModel({
    required this.quantity,
    required this.accommodation,
  });

  factory ReservationAccommodationModel.fromJson(Map<String, dynamic> json) {
    return ReservationAccommodationModel(
      quantity: json['quantity'],
      accommodation: Accommodation.fromJson(json['accommodation']),
    );
  }
}
